tests/lib/test_statedb.py

```diff
@@ -32,13 +32,10 @@
 STATES = list(StateEnum._member_names_)
 
 
-def create_items_bulk(item_count, fns, **kwargs):
-    if callable(fns):
-        fns = [fns]
+def create_items_bulk(item_count, fn, **kwargs):
```

Comment:
Seems like this is confusingly named? Is this more `state_update_bulk`? Whatever, it was here before so don't feel like you have to change it. Just commenting because the name confused me for a moment.